app/views/batch_contexts/_new_bc_form.erb

```diff
@@ -1,5 +1,7 @@
 
-<div data-controller="globus <%= "caption" if Settings.ocr.enabled%>">
+<div data-controller="globus caption"
+     data-caption-stt-enabled="<%=Settings.speech_to_text.enabled%>"
+     data-caption-ocr-enabled="<%=Settings.ocr.enabled%>">
```

Comment:
always enable the caption stimulus controller and always add the controls to the html (but hidden)... we will instead make the settings values visible to the stimulus controller so the JS can do the work of deciding which controls to show/hide as needed based on feature flag settings and user selections

this makes it more flexible and easier to reason with (since it all happens in one place now, in the stimulus controller)